src/Business Foundation/App/NoSeries/src/Upgrade/NoSeriesInstaller.Codeunit.al

```diff
@@ -28,6 +28,7 @@ codeunit 329 "No. Series Installer"
         if UpgradeTag.HasUpgradeTag(NoSeriesUpgradeTags.GetImplementationUpgradeTag()) then
             exit;
 
+        NoSeriesLine.SetRange(Implementation, 0); // Only update the No. Series Lines that are still referencing the default implementation (0)
```

Comment:
Why not use `Enum "No. Series Implementation"`?

Reply:
I want to be explicit it's the default value to any reader, if I write NoSeriesImplementation::Normal you have to know that Normal is the default value by looking it up. Now you can see it right away.